doc/usermanual/darkroom/modules/tone/filmicrgb.xml

```diff
@@ -0,0 +1,619 @@
+      <title>Extreme luminance saturation (<guilabel>Look</guilabel> tab)</title>
+      <para>
+        The darker curve in the graph of the module shows the desaturation of the extremities of
+        the luminance range (black and white): since black and white do not have a color, they
+        should normally be associated to 0% saturation. The default saturation is set to 100% in
+        the range defined by the latitude, and decreases down to 0% outside of that range. One
+        of the advantages of this operations is that, since color components do not clip at the
+        same rate in the image, desaturating them avoids fringes around the high exposures.
+      </para>
+      <para>
+        If the bright colors feel too desaturated, you should check that the white-relative
+        exposure setting does not clip the high luminance spots, and if not, increase the
+        extreme luminance saturation parameter.
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>Preserve chrominance (<guilabel>Look</guilabel> tab)</title>
+      <para>
+        The
+        <guilabel>preserve chrominance</guilabel>
+        setting indicates how the chrominance should be handled by filmic rgb: either not at
+        all, or using one of the provided three norms.
+      </para>
+      <para>
+        When applying the S-curve transformation independently on each color, the proportions of
+        the colors get modified, which modifies the properties of the underlying spectrum, and
+        ultimately the chrominance of the image. This is what happens if you choose
+        <emphasis>no</emphasis> in the
+        <guilabel>preserve chrominance</guilabel>
+        parameter. This value may yield seemingly <quote>better</quote> results than the other
+        values, but it may negatively impact later parts of the pipeline, when it comes to
+        global saturation for example.
+      </para>
+      <para>
+        The other values of this parameter all work in a similar way. Instead of applying the
+        S-curve to the channels R, G and B independently, filmic rgb uses a norm N, divides all
+        the three components by that norm, and applies the S-curve to N. This way, the
+        relationship between the channels is preserved.
+      </para>
+      <para>
+        The different values of the
+        <guilabel>preserve chrominance</guilabel>
+        parameter indicate which norm is used (the value used for N):
+        <itemizedlist>
+
+          <listitem><para>
+            <emphasis>max RGB</emphasis> is the maximum value of the three channels R, G and B.
+            It is the behaviour of the previous version of the filmic module. It tends to darken
+            the blue, especially skies, and to yield halos/fringes, especially if some channels
+            are clipped.
+          </para></listitem>
+
+          <listitem><para>
+            <emphasis>luminance Y</emphasis> is the sum of the three channels R, G and B. It
```

Comment:
not really the sum, it's a linear combination that comes from the XYZ <-> RGB definition.